src/Azure.Functions.Cli/Azure.Functions.Cli.nuspec

```diff
@@ -5,7 +5,7 @@ Licensed under the MIT license. See LICENSE file in the project root for full li
 -->
 <package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
   <metadata>
-    <id>Microsoft.Azure.Functions.CoreTools.V4</id>
+    <id>Microsoft.Azure.Functions.CoreTools</id>
```

Comment:
Can you point to where this nuget package is referenced? I assume there's a few places we'd have to update.

Reply:
This nuget package is included in our releases of Core Tools and was originally named `Microsoft.Azure.Functions.CoreTools` (see [here](https://github.com/Azure/azure-functions-core-tools/releases)) I added the `V4` at the end when creating the `.nuspec` file for SBOM generation, but later realized it was unnecessary since the version already indicates it is V4.